### DISCUSSION TITLE
Are highlighting export tools => Joplin & Readwise disabled or removed in current nightly release (2023.10-55)

### DISCUSSION THREAD

**Discussion by arooni** [koreader/koreader#11241] [General]
The manual says:

Exporting your highlights
There are different ways of exporting your highlights from your reader:
- You can use the Export Highlights plugin to export your notes in text, html, JSON or markdown formats to your device or
export them directly into Joplin (a note taking application) over a network (you can find help about this under Joplin menu in Export
Highlights plugin - https://joplinapp.org/):
TOP MENU > > Export Highlights
- Also you can export the notes to your Readwise account
- You can use an external program named KoHighlights on your computer. It can access your book notes and highlights when you
plug your reader to your computer: https://github.com/noembryo/KoHighlights

But it seems like this doesn't exist in my nightly version.  Am I doing something wrong?  Or does such export need to be installed separately?

**Comment by Frenzie** [comment]
In export highlights there should be something like "choose formats and services"

**Reply by arooni** [reply]
I think I copied the files wrong initially; extracted the zip on the Kindle scribe and copied the backup files to it and it all works now without issue.  VERY COOL export to readwise; WOOT!  one of my favorite features.

**Comment by jonnyl2** [comment]
Also make sure the `Export highlights` plugin is active  in 

Top Menu -> Tools (4th) -> More tools -> Plugin management ->
